packages/content/src/Infrastructure/Sulu/Admin/ContentViewBuilderFactory.php

```diff
@@ -34,46 +37,17 @@
  */
 class ContentViewBuilderFactory implements ContentViewBuilderFactoryInterface
 {
-    /**
-     * @var ViewBuilderFactoryInterface
-     */
-    private $viewBuilderFactory;
-
-    /**
-     * @var PreviewObjectProviderRegistryInterface
-     */
-    private $objectProviderRegistry;
-
-    /**
-     * @var ContentMetadataInspectorInterface
-     */
-    private $contentMetadataInspector;
-
-    /**
-     * @var SecurityCheckerInterface
-     */
-    private $securityChecker;
-
-    /**
-     * @var array<string, array{instanceOf: class-string}>
-     */
-    private $settingsForms;
-
     /**
      * @param array<string, array{instanceOf: class-string}> $settingsForms
      */
     public function __construct(
-        ViewBuilderFactoryInterface $viewBuilderFactory,
-        PreviewObjectProviderRegistryInterface $objectProviderRegistry,
-        ContentMetadataInspectorInterface $contentMetadataInspector,
-        SecurityCheckerInterface $securityChecker,
-        array $settingsForms
+        private ViewBuilderFactoryInterface $viewBuilderFactory,
+        private ActivityViewBuilderFactoryInterface $activityViewBuilderFactory,
```

Comment:
didn't we say its responsoble of page, article and snippet admin to add activity to the insights?